src/changelog/utils.py

```diff
@@ -151,9 +153,9 @@ def bump_version(self, version, release_type):
             z = 0
         else:
             z = int(z) + 1
-        return "{}.{}.{}".format(x, y, z)
+        return f"{x}.{y}.{z}"
 
-    def match_version(self, line):
+    def match_version(self, line: str) -> Union[bool, str]:
```

Comment:
Looking at this function now, and seeing the Union return type, I think we would be better making it 
```
def match_version(self, line: str) -> Optional[str]:
     ...
     return None
```
This would allow us to avoid the `cast` in some of the use cases

Reply:
Done. Removed a use of `cast`.